### PR TITLE
fix-currency-display

### PR DESCRIPTION
**Pull request by molly-tran**

### CODE REVIEW

File: src/app/components/trade/LiquidityDetails.tsx [hunk 1]
```diff
@@ -425,7 +425,12 @@ const PoolRecord = ({
     <>
       <ListItem>
         <StyledDataText>
-          <DataText>{`${aBalance.currency.symbol || '...'} / ${bBalance.currency.symbol || '...'}`}</DataText>
+          {aBalance.currency.symbol == 'bnUSD' ||
```
Comment (shallcdn) [inline src/app/components/trade/LiquidityDetails.tsx:428]: Should use '===' instead of  '=='

**Review by shallcdn — CHANGES_REQUESTED** [review]
Please update as my comment

**Review by shallcdn — APPROVED** [review]
LGTM

**Comment by BennyOptions** [comment]
Hey @molly-tran @shallcdn this should not be a hard-coded solution. There is always a base_currency and quote_currency. Base is on the left, quote is on the right. Something is incorrect in the logic of the app, simple hard coding is not the ideal solution.

**Comment by BennyOptions** [comment]
Also, can you guys set this PR to main net so I can test with my wallets that have liquidity? Otherwise I need to get more test tokens which will take more time.

**Comment by shallcdn** [comment]
> Also, can you guys set this PR to main net so I can test with my wallets that have liquidity? Otherwise I need to get more test tokens which will take more time.

@BennyOptions I have deployed the current change with prod environment, please take a look!

**Comment by BennyOptions** [comment]
> > Hey @molly-tran @shallcdn this should not be a hard-coded solution. There is always a base_currency and quote_currency. Base is on the left, quote is on the right. Something is incorrect in the logic of the app, simple hard coding is not the ideal solution, because it will display new pools incorrectly.
> 
> Hi @BennyOptions Where can I check the correct order of the pool pairs? on this page, https://stats.balanced.network/ Am I correct?

Yes that's correct.

Also, as a rule, sICX is always on the right side (except for sICX/bnUSD). bnUSD is also always on the right side (never any exceptions)

**Comment by BennyOptions** [comment]
The "your supply" column is still backwards.

Screen Shot 2022-08-10 at 2 57 27 PM

**Comment by BennyOptions** [comment]
One thing that confuses me, @shallcdn do you know why sICX/bnUSD is correct, but all others are incorrect?

**Comment by shallcdn** [comment]
> One thing that confuses me, @shallcdn do you know why sICX/bnUSD is correct, but all others are incorrect?
@BennyOptions 
I have investigated this issue, there is a logic to checking and comparing between two addresses of the base token and quote token to create a pair, 

Eg: Suppose that we have sICX and BALN
sICX address (): 'abd' 
BALN address: 'abc'

The logic will compare 'abd' < 'abc', it will return `false` and create a pair is BALN/sICX, if `true` the pair will be sICX/BALN. 

[JavaScript uses the so-called "dictionary" or "lexicographical" order.](https://javascript.info/comparison#string-comparison) that's why 'abd' is greater than 'abc'

I'm not sure why we used this logic, @octocat512 @hetfly. Do you know why we use this logic to create a Pair?

**Comment by shallcdn** [comment]
PR https://github.com/balancednetwork/balanced-network-interface/pull/1061 fixed issue #1050 so I close this PR

**Pull request closed without merging**